app/javascript/components/recipes/index/base-recipe.vue

```diff
@@ -17,5 +17,15 @@ export default {
   props: {
     recipe: { type: Object, required: true },
   },
+  methods: {
+    ingredientFinalPrice(quantity, price) {
+      return (quantity * price);
+    },
+  },
+  computed: {
+    getPrice() {
+      return this.recipe.ingredients.map(element => this.ingredientFinalPrice(element.quantity, element.price)).reduce((acc, curVal) => acc + curVal, 0);
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This line has a length of 153. Maximum allowed is 120  vue/max-len